src/Tasks/Microsoft.NET.Build.Tasks/PreprocessPackageDependenciesDesignTime.cs

```diff
@@ -236,7 +236,7 @@ private DependencyType GetDependencyType(string dependencyTypeString)
 
                 var currentPackageUniqueId = $"{parentTargetId}/{currentItemId}";
                 // add current package to dependencies world
-                var currentItem = items[currentItemId];
```

Comment:
Just pushed a change for that

---
In reply to: [113310902](https://github.com/dotnet/sdk/pull/1149#discussion_r113310902) [](ancestors = 113310902)